apps/summon-app/index.html

```diff
@@ -8,6 +8,11 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="/src/main.tsx"></script>
+    <script>
```

Comment:
The only thing that worries me about this is that we need all clients using this package to add this to their app UI.

If this is the only way to solve - then maybe we need to get some 'Getting Started' docs for this package and ensure this is communicated. Thoughts @jonathanprozzi ?

Reply:
Yes, it will be necessary for all vite apps. We should document it. Let me know where and I can add something

Reply:
oh ok. just for vite. cool that alleviate my concern.

Reply:
I was wondering that too since this is within our package. We can be sure to add this into the docs